src/Autocomplete.js

```diff
@@ -123,20 +200,39 @@ class Autocomplete extends AutocompleteBase {
 	}
 
 	/**
+	 * Adds the CSS class to mark that item as selected.
+	 * @protected
+	 */
+	selectItem_(option) {
+		dom.addClasses(option, 'active');
+	}
+
+	/**
 	 * Emits a `select` event with the information about the selected item and
-	 * hides the element.
-	 * @param {!Element} item The list selected item.
+	 * hides the list element.
+	 * @param {!Object} item The list selected item.
 	 * @protected
 	 */
-	onListItemSelected_(item) {
-		var selectedIndex = parseInt(item.getAttribute('data-index'), 10);
+	selectOption_(selectedItem) {
```

Comment:
Yes, I agree. I've used activeListItem_() to make clear that we are talking about an item of the List component.